lntest/itest/lnd_neutrino_test.go

```diff
@@ -0,0 +1,45 @@
+package itest
+
+import (
+	"context"
+
+	"github.com/lightningnetwork/lnd/lnrpc/neutrinorpc"
+	"github.com/lightningnetwork/lnd/lntest"
+	"github.com/stretchr/testify/require"
+)
+
+// testNeutrino checks that the neutrino sub-server can fetch compact
+// block filters, server status and connect to a connected peer.
+func testNeutrino(net *lntest.NetworkHarness, t *harnessTest) {
+	if net.BackendCfg.Name() != lntest.NeutrinoBackendName {
+		t.Skipf("skipping test for non neutrino backends")
+	}
+
+	ctxb := context.Background()
+
+	// Check if the neutrino sub server is running.
+	statusReq := &neutrinorpc.StatusRequest{}
+	ctxt, _ := context.WithTimeout(ctxb, defaultTimeout)
+	statusRes, err := net.Alice.NeutrinoClient.Status(ctxt, statusReq)
+	require.NoError(t.t, err, "unable to receive the status from the "+
```

Comment:
I'm personally not a huge fan of adding a lot of text as the last argument of `require.XXX()` calls. Usually the context of where the error happens (pretty obvious from the generated error stack trace that the `require` library produces) and the error message itself provides enough context. And if the text causes the line to become too long, this actually deteriorates readability. So I'd only add the additional context for `require.True` calls where you really don't have any other context.

Reply:
Makes sense.